src/chericat.c

```diff
@@ -55,25 +55,30 @@
 
 static void usage()
 {
-	fprintf(stderr, "Usage: chericat [-d <debug level>] [-f <database name>] [-p <pid>] [-v]\n\t[-c <binary name>]\n"
-			"     debug level - 0 = No output; 1 = INFO; 2 = VERBOSE; 3 = TROUBLESHOOT\n"
-			"     pid - pid of the target process for a snapshot of caps info\n"
+	fprintf(stderr, "Usage: chericat [-d*] [-f <database name>] [-p <pid>] [-v]\n\t[-c <binary name>]\n"
+			"     pid           - pid of the target process for a snapshot of caps info\n"
 			"     database name - name of the database to store data captured by chericat\n"
+			"     binary name   - name of the binary file for which to show the\n"
+			"                     capabilities located, with corresponding symbols\n"
 			"Options:\n"
-			"     -d Determine the level of debugging messages to be printed. If omitted,\n"
-			"        the default is INFO level\n"
+			"     -d The number of 'd' provided determines the level of debugging messages\n"
+		        "        to be printed. If omitted, the default is NO_PRINT level, meaning no\n"
+			"        debugging output. If one 'd' (i.e. -d) is provided, the level is set\n"
+			"        to INFO. If two 'd' (i.e. -dd) is provided, the level is set to VERBOSE.\n"
+			"        The most verbose level is three 'd' (i.e. -ddd), which is the\n"
+			"        TROUBLESHOOTING level. Any more 'd' after three will be ignored.\n" 
 			"     -f Provide the database name to capture the data collected by chericat.\n"
```

Comment:
Arguably too much detail for usage. Naming the levels isn't particularly informative.